samples/apps/copilot-chat-app/scripts/start-first-run.cmd

```diff
@@ -0,0 +1 @@
+.\1_install-reqs.cmd && start cmd.exe /k ".\2_init-backend.cmd %1" & start cmd.exe /k ".\3_init-frontend.cmd %2 %3"
```

Comment:
I would consider not having a "first run" specific case. Dependency installation should be idempotent - let's try and keep it simple with just "start.cmd" that either includes "install-reqs" or have the user call install-reqs via instructions in README.